src/Sdk/DTPipelines/Pipelines/AgentJobRequestMessage.cs

```diff
@@ -184,6 +184,20 @@ public WorkspaceOptions Workspace
             private set;
         }
 
+        [DataMember(EmitDefaultValue = false)]
```

Comment:
Maybe we need to update the initialization method `AgentJobRequestMessage` as well? I think this should match with the definition in Actions Service, so maybe it is not manually updated but somehow updated with dependency. I am not sure what is the best practice here.

Reply:
It's not necessary since this class is only initialized through deserialization but I've added the constructor mapping for completeness.

Reply:
I've removed it since it changes the constructor signature and causes some tests to break as the constructor signature no longer matches.